packages/trpc/server/routers/viewer.tsx

```diff
@@ -1142,6 +1142,17 @@ const loggedInViewerRouter = router({
       return prev - (current._count?.recurringEventId - 1);
     }, count);
   }),
+  isTeamsPlan: authedProcedure.query(async ({ ctx }) => {
+    const user = await ctx.prisma.user.findFirst({
+      where: {
+        id: ctx.user.id,
+      },
+      select: {
+        teams: true,
+      },
+    });
+    return (user && user.teams.length > 0) as boolean;
```

Comment:
```suggestion
    const membershipCount = await ctx.prisma.membership.count({
      where: { userId: ctx.user.id },
    });
    return membershipCount > 0;
```

We only need to count memberships